packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@builder.io/sdk",
-  "version": "3.0.1",
+  "version": "3.0.2-2",
```

Comment:
for my knowledge, what does the `-2` signify here?

Reply:
That's a dev release

Reply:
is it a common thing for us to have core packages depend on dev versions of the sdk?

Reply:
@teleaziz do you think we should add this? Because after we merge this PR, changeset bot will automatically bump versions and add a patch version and update that in our React gen1 SDK. I think we can keep it `3.0.1` here and for React gen1 - `5.0.1` as it will auto bump it to `3.0.2` and `5.0.2` respectively and link that to the gen1 React SDK (`5.0.2`). With this change, it might skip `3.0.2` and create `3.0.3` and I think it would've also updated yarn.lock here? We can skip the changesets if we are planning to only dev-release it

Reply:
@kylefowler It's not common or recommended for core packages to depend on dev versions of the SDK. In this specific PR, nothing is actually depending on a dev version of another library. The change to version "3.0.2-2" is intended to test the core SDK standalone.

Using a dev version (like "3.0.2-2") for the SDK itself is a good practice for testing changes before a final release. However, other packages, especially core ones, typically shouldn't rely on these unstable dev versions.

The purpose here is to validate the SDK changes in isolation, ensuring they work as expected.


@sidmohanty11 The correct sequence would be from "3.0.2-2" to the final "3.0.2" release, not to "3.0.3". In semantic versioning, the "-2" suffix indicates a pre-release version. When finalizing, this suffix is removed to create the stable release. The version would only increment to 3.0.3 if additional changes were made after the 3.0.2 release

Reply:
@teleaziz thanks for explaining the semantic sequencing concept! Makes sense. stupid me, i thought changeset bot will +1 it.